doc/man3/OSSL_trace_set_channel.pod

```diff
@@ -128,12 +128,11 @@ trace hook is set.
 
 Traces OpenSSL library initialization and cleanup.
 
-This needs special care, as OpenSSL will do automatic cleanup after
-exit from C<main()>, and any tracing output done during this cleanup
-will be lost if the tracing channel or callback were cleaned away
-prematurely.
-A suggestion is to make such cleanup part of a function that's
-registered very early with L<atexit(3)>.
+Keep in mind the OpenSSL cleanup function (OPENSSL_cleanup()) is
+invoked by library destructor. This is typically arranged by linker.
+So if your function is tracing OSSL_TRACE_CATEGORY_INIT category,
+then file descriptors to log files (log facilities) must be left
+open after the C<main()> exits.
```

Comment:
delete `the`

Reply:
This comment from t8m is still outstanding